src/controllers/Controller.java

```diff
@@ -5,66 +5,150 @@
 import javafx.event.EventHandler;
 import javafx.fxml.FXML;
 import javafx.fxml.Initializable;
+import javafx.scene.Group;
+import javafx.scene.Scene;
 import javafx.scene.control.ListView;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
 import javafx.scene.input.KeyCode;
 import javafx.scene.input.KeyEvent;
+import javafx.scene.input.MouseEvent;
 import javafx.scene.layout.Pane;
 import models.CategoryModel;
+import models.FoodItemModel;
 import services.CategoriesService;
-
+import services.FoodItemsService;
 import java.net.URL;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.ResourceBundle;
 
 public class Controller implements Initializable {
-    @FXML
-    ListView categoryListView;
+    private HashMap<String, CategoryModel> categoryModelHashMap;
 
     @FXML
-    Pane backPane;
+    public ListView categoryListView;
+    public ListView foodItemsListView;
+    public Pane backPane;
+    public TableView<FoodItemModel> receiptTableView;
+    private ObservableList<FoodItemModel> selectedFoodItemsToDisplay = FXCollections.observableArrayList();
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
 
-        populateListView();
-
+        CategoriesService categoriesService = new CategoriesService();
+        FoodItemsService foodItemsService = new FoodItemsService();
+        categoryModelHashMap = categoriesService.get();
+        setupTableColumns();
+        initializeReceiptTable();
+        initializeEventListeners(categoryModelHashMap);
```

Comment:
❌ `initializeReceiptTable()` is really initializing an event handler and should really be placed in the `initializeEventListeners()` routine.